.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns:
+          - "*"
+    labels:
+      - "github-actions"
+      - "dependencies"
+    reviewers:
+      - "BenGalewsky"
+      - "ponyisi"
```

Comment:
This automatically assigns @BenGalewsky and @ponyisi as reviewers. If alternate maintainers should be assigned we can change that.